### PR TITLE
Save magnetic values and update doc

### PR DESCRIPTION
**Pull request by cua-cua**

### CODE REVIEW

File: capture.py [hunk 1]
```diff
@@ -83,12 +90,10 @@ def get_sensor_values(i2c):
         break
     return reply, readings, ack, i, pd0, frame_counter
 
-
```
Comment (Peque) [inline capture.py]: Why did you remove this line?

File: capture.py [hunk 2]
```diff
@@ -83,12 +90,10 @@ def get_sensor_values(i2c):
...
 class Node:
     def __init__(self):
         """
         Initialize the node.
-
```
Comment (Peque) [inline capture.py]: Why did you remove this line?

File: capture.py [hunk 3]
```diff
@@ -160,12 +165,31 @@ def update_savings(self, savings):
         reply = urequests.get(url)
         return reply
 
+    def read_coin(self):
```
Comment (Peque) [inline capture.py]: Could you add a very short docstring? :blush:

File: capture.py [hunk 4]
```diff
@@ -160,12 +165,31 @@ def update_savings(self, savings):
...
+        self.led.on()
+        for i in range (num_measures):
+            reply, (x, y, z, t), ack = read_sensor(self.i2c)
+            x_array[i] = x
+            y_array[i] = y
+            z_array[i] = z
+        self.led.off()
+    
+    def save_readings(self):
```
Comment (Peque) [inline capture.py]: Could you add a very short docstring? :blush:

File: capture.py [hunk 5]
```diff
@@ -160,12 +165,31 @@ def update_savings(self, savings):
...
+        with open('readings.csv', 'w') as f:
+            for i in range (num_measures):
+                line = '{x},{y},{z}\n'.format(
```
Comment (Peque) [inline capture.py]: Maybe we could add column names before the loop?

In example: `f.write('x,y,z\n')`

File: docs/source/index.rst [hunk 6]
```diff
@@ -28,12 +28,14 @@ application.
...
-How can I build it?
--------------------
+Index
+-----
 
 .. toctree::
    :maxdepth: 2
 
+   introduction.rst
+
```
Comment (Peque) [inline docs/source/index.rst]: I would not add a blank line between sections.

File: docs/source/introduction.rst [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+In order to design our coin box we will take into account euro coin characterisitcs. 
+
+- The maximum diameter is 25.75 mm for 2 euro coin
+- The minimum diameter is 16.25 mm for 1 cent coin
+- The maximum edge thickness is 2.38 mm for 50 cent
+- The minimum edge thickness is 1.67 mm for 1, 2 and 5 cents
+
+About composition:
+
+- €2: outer ring - cupronickel; inner section (three layers) - nickel-brass, nickel, nickel-brass (slightly magnetic)
```
Comment (Peque) [inline docs/source/introduction.rst]: 2 €

File: docs/source/introduction.rst [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+- €1: outer ring - nickel-nickel, inner section (three layers) - cupronickel, nickel, cupronickel (slightly magnetic)
+- 50 cent, 20 cent and 10 cent: copper, aluminium, zinc, tin alloy = "Nordic gold" (non-magnetic)
+- 5, 2 and 1 cent: copper-coated steel (highly magnetic)
+
+3D 2GO board
+------------
```
Comment (Peque) [inline docs/source/introduction.rst]: Remove this section? (the image is not even related)

File: docs/source/introduction.rst [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+- The minimum diameter is 16.25 mm for 1 cent coin
+- The maximum edge thickness is 2.38 mm for 50 cent
+- The minimum edge thickness is 1.67 mm for 1, 2 and 5 cents
+
+About composition:
+
+- €2: outer ring - cupronickel; inner section (three layers) - nickel-brass, nickel, nickel-brass (slightly magnetic)
+- €1: outer ring - nickel-nickel, inner section (three layers) - cupronickel, nickel, cupronickel (slightly magnetic)
+- 50 cent, 20 cent and 10 cent: copper, aluminium, zinc, tin alloy = "Nordic gold" (non-magnetic)
+- 5, 2 and 1 cent: copper-coated steel (highly magnetic)
```
Comment (Peque) [inline docs/source/introduction.rst]: I'd rather use sublists or say "The outer ring is composed... . The inner section has three layers, the first..., the second...".

File: capture.py (hunk 1 above)
Comment (cua-cua) [reply capture.py]: I added it again, I didn't see it during merge

File: capture.py (hunk 2 above)
Comment (cua-cua) [reply capture.py]: I added it again, I didn't see it during merge

File: docs/source/index.rst (hunk 6 above)
Comment (cua-cua) [reply docs/source/index.rst]: new pull request for documentation

File: docs/source/introduction.rst (hunk 7 above)
Comment (cua-cua) [reply docs/source/introduction.rst]: new pull request for documentation

File: docs/source/introduction.rst (hunk 8 above)
Comment (cua-cua) [reply docs/source/introduction.rst]: new pull request for documentation

File: docs/source/introduction.rst (hunk 9 above)
Comment (cua-cua) [reply docs/source/introduction.rst]: new pull request for documentation
